src/backend/src/controllers/description-Bullets.controllers.ts

```diff
@@ -0,0 +1,14 @@
+import { Request, Response } from 'express';
+import { validationResult } from 'express-validator';
+
+export const checkWorkPackageDescriptionBullet = async (req: Request, res: Response) => {
+  const errors = validationResult(req);
+  if (!errors.isEmpty()) {
+    return res.status(400).json({ errors: errors.array() });
+  }
+
+  const { body } = req;
+  const { submitterId, descriptionBulletId } = body;
+
+  return res.status(200).json(body);
```

Comment:
so the set up of this endpoint is correct but you're just missing the functionality of it. What you need to do is have the endpoint use the submitterId and descriptionBulletId to set the `userCheckedId` value in prisma to be the submitterId of the bullet with that id.

Idk if that makes total sense but this endpoint https://github.com/Northeastern-Electric-Racing/FinishLine/pull/145 is very similar but uses risks instead of description bullets. also the schema.prisma file will help you a lot to see what the data model is like.

oh also can you do the exact same permission checks that the edit risks uses pls